app/sdil/config/FrontendAppConfig.scala

```diff
@@ -34,8 +34,8 @@ trait AppConfig {
   val directDebitEnabled: Boolean
 }
 
-class FrontendAppConfig(val runModeConfiguration: Configuration, runMode: RunMode, environment: Environment)
-    extends ServicesConfig(runModeConfiguration, runMode) with AppConfig {
+class FrontendAppConfig(val runModeConfiguration: Configuration, environment: Environment)
```

Comment:
This shouldn't be called runModeConfiguration, runMode is no longer relevant.  Perhaps it can be called just configuration, assuming that is available here